homeassistant/components/izone/climate.py

```diff
@@ -54,6 +58,18 @@
     Controller.Fan.AUTO: FAN_AUTO,
 }
 
+ATTR_AIRFLOW = "airflow"
+
+IZONE_SERVICE_AIRFLOW_MIN = "airflow_min"
+IZONE_SERVICE_AIRFLOW_MAX = "airflow_max"
+
+IZONE_SERVICE_AIRFLOW_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_ENTITY_ID): cv.entity_id,
+        vol.Required(ATTR_AIRFLOW): cv.positive_int,
```

Comment:
I could be wrong, but I think there's range validators available, plus validators that round to the closest 5.

Reply:
I found the range validator(min=0, max=100) and will add that however I can't see any rounding stuff. Can you link an example please?

Reply:
I don't know, it might not exist.

Reply:
Ok. I've had a look and couldn't find anything. I've added the range validation so might mark this item as resolved.